molr-mole-core/src/main/java/io/molr/mole/core/runnable/lang/ctx/OngoingContextualBranchWithNewContext.java

```diff
@@ -43,4 +44,20 @@ public void as(BiConsumer<ContextualBranch<C>,Placeholder<C>> branchDescription)
         ContextualBranch<C> branch = new ContextualBranch<>(builder(), block, contextPlaceholder);
         branchDescription.accept(branch, contextPlaceholder);
     }
+
+    public void as(Consumer<ContextualBranch<C>> branchDescription) {
```

Comment:
Looks a bit like copy paste here ... shouldn't this method just delegate to the upper one? Something like:
``` 
public void as(Consumer<ContextualBranch<C>> branchDescription) {
   as( (branch,ctx) -> branchDescription.accept(branch));
}
```
?